src/ui/pages/Popup/popup.scss

```
@@ -44,12 +45,12 @@ a {
   }
 
   .popup {
-    width: 36rem;
+    width: 100%;
```

When popup is opened as a page, then 100% takes up the full page width. If 36rem is causing issues somewhere else, then how about:

```suggestion
    max-width: 36rem;
```

---

yes. i tried to implement a nice new side-bar feature, but it ended up being a bit more complex than anticipated